<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>146</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>681</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white </string>
    </property>
    <property name="text">
     <string>Mars(Марс, Марсик)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white </string>
    </property>
    <property name="text">
     <string>Pos 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white </string>
    </property>
    <property name="text">
     <string>Pos 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-image: url(reMars.png);
height: 140px;
border-radius: 15px;
color: white </string>
    </property>
    <property name="text">
     <string>Pos 3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>80</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-image: url(Tiny.png);
height: 140px;
border-radius: 15px;
color: white </string>
    </property>
    <property name="text">
     <string>Pos 4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-image: url(Clockwerk);
height: 140px;
border-radius: 15px;
color: white </string>
    </property>
    <property name="text">
     <string>Pos 5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>971</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>971</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: rgb(50, 50, 50);
height: 140px;
border-radius: 15px;
color: white </string>
    </property>
    <property name="text">
     <string>Баратрум - гордое и могущественное существо, яростный первичный разум. В мир материи он пришёл, чтоб участвовать в событиях, отзвуки которых донесутся до его родной первичной вселенной. Сам собрал своё тело Баратрум, и послужит оно ему как в нашем мире, так и за его пределами. Его
бренная оболочка черпает силы из нашей
вселенной, объединяет она формы быка и
обезьяны: рога, копыта и лапы воплощают его силу, скорость и ловкость. Кольцо в носу
напоминает Баратруму, что служит он
тайному господину и что мир, где трудится
он, - лишь тень настоящего бытия.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Talents">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-image: url(Tree.png);
height: 140px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 100)</string>
    </property>
    <property name="text">
     <string>Talents</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Skill1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-image: url(ChargeOfDarkness.png);
height: 140px;
border-radius: 15px;
border: 1px solid rgba(255, 255, 255, 100)</string>
    </property>
    <property name="text">
     <string>Skill1</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Skill2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-image: url(Bulldoze.png);
height: 140px;
border-radius: 15px;
border: 1px solid rgba(255, 255, 255, 100)</string>
    </property>
    <property name="text">
     <string>Skill2</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Skill3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: red;
height: 140px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 100)</string>
    </property>
    <property name="text">
     <string>Skill3</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Skill4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">width: 255px;
background-color: red;
height: 140px;
border-radius: 15px;
color: white;
border: 1px solid rgba(255, 255, 255, 100)</string>
    </property>
    <property name="text">
     <string>Skill4</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="TalentsInfo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>280</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Skill1Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>280</width>
      <height>350</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Skill2Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>279</width>
      <height>315</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Skill3Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>260</y>
      <width>250</width>
      <height>322</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Skill4Info">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>258</width>
      <height>340</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 20px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="hide" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
